cmd/ztest/ztest.c

```diff
@@ -5644,18 +5644,26 @@ ztest_fletcher(ztest_ds_t *zd, uint64_t id)
 	while (gethrtime() <= end) {
 		int run_count = 100;
 		void *buf;
+		struct abd *abd_data, *abd_meta;
 		uint32_t size;
+
```

Comment:
[cstyle] We don't want extra white space in the variable declarations.  Please remove.